src/browser/rollbar.js

```diff
@@ -371,6 +399,14 @@ Rollbar.prototype._createItem = function(args) {
   return item;
 };
 
+Rollbar.prototype._sameAsLastError = function(item) {
+  if (this.lastError === item.err && this.lastError) {
```

Comment:
Should these two conditions be swapped? I imagine the second is more common than the first.